styles.css

```diff
@@ -1 +1,247 @@
-html { margin: 0; }
+* {
+  margin: 0%;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+  align-items: center;
+}
+
+.fisherman {
+  max-width: 100%;
+  height: 100vh;
+  max-height: 576px;
+  background-image: url('images/Rectangle 1 (1).png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center;
+  padding: 0 2em;
+}
+
+h1 {
+  padding: 0 3%;
+  padding-top: 4.8%;
+  font-family: Bebas Neue, Open Sans, sans-serif;
+  font-size: 64px;
+  font-weight: 400;
+  line-height: 77px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  justify-content: center;
+  margin-top: 0%;
+}
+
+p {
+  padding: 0 4%;
+  padding-top: 12%;
+  font-family: Poppins, Open Sans, sans-serif;
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 36px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+}
+
+.content {
+  max-width: 100%;
+  height: 100vh;
+  max-height: 563px;
+  justify-content: center;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  padding: 0 2em;
+}
+
+.fish-hook1 > h2 {
+  font-family: Poppins, Open Sans, sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 27px;
+  letter-spacing: 0;
+  text-align: left;
+  margin: 14.94px 0;
+}
+
+.fish-hook1 > input[type="name"] {
```

Comment:
Remember DRY, you can use some selectors that have the same rules.
lines 73-79 and 93-99 have the same properties.

.fish-hook1 > input[type="name"],
.fish-hook2 > input[type="email"] {
   properties in common
}